vue/components/ui/molecules/lightbox/lightbox.vue

```diff
@@ -40,6 +40,10 @@
     width: inherit;
 }
 
+.lightbox.fit > img {
+    object-fit: contain;

```

Comment:
This seems like a breaking change so let's avoid it (specify it on the client instead)